modules/admin_manual/pages/maintenance/package_upgrade.adoc

```diff
@@ -1,21 +1,30 @@
 = Upgrade ownCloud From Packages
 :toc: right
+:packages-url: https://owncloud.com/older-versions/#server
 
-== Upgrade Quickstart
+== Upgrade Steps
 
 The alternative to a manual upgrade is configuring your system to use ownCloud’s
 https://download.owncloud.org/download/repositories/stable/owncloud/[Open Build Service] repository.
```

Comment:
This is just a reminder for later
`https://download.owncloud.org` cant be easily renamed to a `.com` address as it is with some sub-paths physically not present ! Needs to be evaluated to get rid of.